chapters/develop-organize/reading/reading/README.md

```diff
@@ -18,8 +18,31 @@ To enforce the guidelines we use [automated linter checks](https://github.com/su
 Besides the mentioned guidelines, the linter implements checks for other minor style conventions (such as how to define lists, where should new lines be placed and others).
 However, for the majority of cases, the above should be sufficient.
 
+The reading content type is stored as a folder named `reading` in the course repository.
+Each topic should contain a `reading` folder that includes one or multiple Markdown files, depending on how the topic may be broken down in independent reading parts.
```

Comment:
```suggestion
The reading content type is stored as a directory named `reading` in the course repository.
Each topic should contain a `reading` directory that includes one or multiple Markdown files, depending on how the topic may be broken down into independent reading parts.
```
Use "directory" instead of "folder" as stated [here](https://discord.com/channels/926757749763756093/1123983780328247326/1141287344301932626).